website/content/en/docs/cloud-providers/AWS/launch-templates.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Launch Templates and Custom Images"
+linkTitle: "Launch Templates"
+weight: 80
+---
+
+By default, Karpenter generates launch templates that use [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) for nodes. Often, users need to customize the node image to integrate with existing infrastructure or meet compliance requirements. Karpenter supports custom node images through Launch Templates. If you need to customize the node, then you need a custom launch template. 
+
+Note: By customizing the image, you are taking responsibility for maintianing the image, including security updates. In the default configuration, Karpenter will use the latest version of the EKS optimized AMI, which is maintained by AWS. 
+
+
+## Introduction
+
+Karpenter follows existing AWS patterns for customizing the base image of
+instances. More specifically, Karpenter uses [EC2 launch templates](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html). Launch
+templates may specify many values. The pivotal value is the base image (AMI).
+Launch templates further specify many different parameters related to networking, authorization, instance type, and more. 
+
+**Karpenter only implementes a subset of launch template fields, and some fields should not be set.**  
+
+This guide describes requirements for using launch templates with Karpenter, and later an example procedure.
+
+## Launch Template Requirements
+
+The Launch Template resource includes a large number of fields. AWS accepts launch templates with any subset of these fields defined.
+
+Certain fields are obviously critical, such as AMI and User Data. Some fields are useful for particular workloads, such as storage and IAM Instance Profile. 
+
+Finally, **the majority of Launch Template fields should not be set** (or will have no effect), such as network interfaces and instance type. 
+
+## Important Fields
+
+When creating a custom launch template, the AMI and User Data are the defining characteristics. Instance Profile (IAM Role) and Security Group (firewall rules) are also important for Karpenter. 
+
+### AMI
+
+AMI (Amazon Machine Image), is the base image/VM for a launch template.
+
+[Review the instructions for importing a VM to AWS.](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) Note the AMI id generated by this process, such as, 
+`ami-074cce78125f09d61`.
+
+### User Data - Autoconfigure
+
+Importantly, the AMI must support automatically connecting to a cluster based
+on "user data", or a base64 encoded string passed to the instance at startup.
+The syntax and purpose of the user data varies between images. The Karpenter
+default OS, Amazon Linux 2 (AL2), accepts shell scripts (bash commands). 
+
+[AWS calls data passed to an instance at launch time "user
+data".](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-shell-scripts)

```

Comment:
Feels like this should go before the previous paragraph. Either the reader understood this from the previous paragraph, or they didn't and that previous paragraph confused them. IMHO, either we should remove this or put it first.

Reply:
I'm happy with the first sentence.

> Importantly, the AMI must support automatically connecting to a cluster based
on "user data", or a base64 encoded string passed to the instance at startup.

There is no graceful way to introduce these confusing terms.